src/components/Skills/index.js

```diff
@@ -1,7 +1,30 @@
-import React from 'react'
+import styles from './Skills.module.css'
+import { FaHtml5, FaCss3, FaReact, FaNode } from 'react-icons/fa'
+import { TbBrandJavascript } from 'react-icons/tb'
+import {
+    SiMysql,
+    SiMongodb,
+    SiApollographql,
+    SiTailwindcss
+} from 'react-icons/si'
 
 function Skills() {
-    return <div>Skills</div>
+    return (
+        <body>
+            <h3 className={styles.skillsheader}>Familiar Technologies</h3>
+            <section className={styles.skillscontainer}>
+                <FaHtml5 className={styles.skillsicon} size={100} />
+                <FaCss3 className={styles.skillsicon} size={100} />
+                <TbBrandJavascript className={styles.skillsicon} size={100} />
+                <FaReact className={styles.skillsicon} size={100} />
+                <FaNode className={styles.skillsicon} size={100} />
+                <SiMysql className={styles.skillsicon} size={100} />
+                <SiMongodb className={styles.skillsicon} size={100} />
+                <SiApollographql className={styles.skillsicon} size={100} />
+                <SiTailwindcss className={styles.skillsicon} size={100} />
```

Comment:
So this is more of an FYI. If you have a section with `.skillsContainer`, you can then just do 
```
.skillsContainer svg {
   apply styles here to all icons at once
}
```

Reply:
Also, lose the `size={100}` here. this is effectively the same thing as inline styling. Apply the size in css to the svg as a `font-size`. That way it's also all controlled by one single point. If you ever had to come change the sizes in this way, you'd have to change 9 lines instead of 1.